src/qt/glcanvas.cpp

```diff
@@ -425,14 +438,26 @@ bool wxGLCanvas::ConvertWXAttrsToQtGL(const int *wxattrs, QGLFormat &format)
 {
     if (!wxattrs)
         return true;
-    return true;
+    //return true;
 
     // set default parameters to false
     format.setDoubleBuffer(false);
     format.setDepth(false);
     format.setAlpha(false);
     format.setStencil(false);
 
+    // Enable multisampling aka MSAA
+
+    format.setSampleBuffers( true );
+    format.setSamples( 4 );
```

Comment:
This is normally defined by the attributes used, why should we hard code this here?

Reply:
I'm still not sure about this one, could you please check it?

Reply:
I am checking.  Will take a little time to find the answers to these two questions.  In mean time I try to clean up all the style and formatting issues.